src/main/resources/META-INF/rewrite/assertj.yml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2020 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.testing.AssertJUseAssertThatFromAssertionsPackage
+recipeList:
+  - org.openrewrite.java.ChangeMethodTargetToStatic:
+      methodPattern: "org.assertj.core.api.AssertionsForClassTypes assertThat(..)"
+      fullyQualifiedTargetTypeName: "org.assertj.core.api.Assertions"
+  - org.openrewrite.java.ChangeMethodTargetToStatic:
+      methodPattern: "org.assertj.core.api.AssertionsForInterfaceTypes assertThat(..)"
+      fullyQualifiedTargetTypeName: "org.assertj.core.api.Assertions"
```

Comment:
Yaml files can include multiple yaml documents (a new yaml document begins wherever you have that non-indented `---` line). I suggest unifying both assertj yamls into assertj.yml.